app/assets/stylesheets/govuk_publishing_components/components/_cookie-banner.scss

```diff
@@ -24,3 +26,70 @@ $govuk-cookie-banner-background: #d5e8f3;
     display: inline-block;
   }
 }
+
+// Styles for the new version of the cookie banner
+.gem-c-cookie-banner--new {
+  @include govuk-font($size: 16);
+  border: 2px solid govuk-colour("black");
+}
+
+.gem-c-cookie-banner__wrapper.js-hidden {
+  display: none;
+}
+
+.gem-c-cookie-banner--new .gem-c-cookie-banner__message {
+  display: inline-block;
+  margin-bottom: govuk-spacing(1);
+  padding-right: govuk-spacing(4);
+  padding-bottom: govuk-spacing(2);
+}
+
+.gem-c-cookie-banner--new .gem-c-cookie-banner__buttons {
```

Comment:
Since the `gem-c-cookie-banner__buttons` and `gem-c-cookie-banner__confirmation` are only being used within the new cookie banner I would avoid chaining these classes.